bot/exts/info/information.py

```diff
@@ -72,7 +72,8 @@ def get_extended_server_info(self, ctx: Context) -> str:
         """Return additional server info only visible in moderation channels."""
         talentpool_info = ""
         if cog := self.bot.get_cog("Talentpool"):
-            talentpool_info = f"Nominated: {len(cog.cache)}\n"
+            num_nominated = len(cog.cache) if cog.cache else 0
```

Comment:
As it stands if the cache is not ready it will say `0`. I'd rather it gave a non-numerical value like `-` to show that, since 0 nominees is possible.

Reply:
Very true, shall fix

Reply:
https://github.com/python-discord/bot/pull/1813/commits/4e61411d2be105ad1e68fa6211d3c95fc3d92b3b